src/services/symbolDisplay.ts

```diff
@@ -730,7 +732,7 @@ function getSymbolDisplayPartsDocumentationAndSymbolKindWorker(typeChecker: Type
         }
     }
 
-    if (tags.length === 0 && !hasMultipleSignatures) {
+    if (tags.length === 0 && !hasMultipleSignatures && !findAncestor(location, isJSDoc)) {

```

Comment:
Seems like this could just be:

```suggestion
    if (tags.length === 0 && !hasMultipleSignatures && !isInJSDoc(location)) {
```

So we don't walk all the way up when we already have a flag for it on the node?